### ISSUE TITLE
Toe contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9690]
**Preferred term label:**

Shahrzad and Noemi, how do the suggestions for new Toe contracture terms relate to these existing terms?
[Camptodactyly of toe HP:0001836](https://hpo.jax.org/app/browse/term/HP:0001836)
Do we need to add extension terms only? For which kinds of toe joint?

**Comment by shahrzadnema** [comment]
The new suggestion for toe consists of Contracture of toe (as a general term)==>Extension contracture of toe. 
Camptodactyly of toe (HP:0001836) is included in the "flexion contracture of toe" (HP:0005830) and retains its' current hierarchy as a member of flexion tree. 
Since flexion contracture (of toe) already exists, we suggest "extension contracture of toe". Our current suggestions do not address for different toes/toe joints, as per our last discussion the differentiation of toe(s)/toe joints are not required in an AMC registry.  
The ideal would be: Contractures of toe (HP:....)==> (1) Extension Contracture of toe (HP:....); (2) Flexion contractures of toe (HP:HP:0005830) (with the existing descendants in the hierarchy).

**Comment by pnrobinson** [comment]
@shahrzadnema 
added the two terms as suggested, thanks!
